README.adoc

```diff
@@ -286,6 +324,11 @@ All Asciidoctor tasks will have the following methods and properties:
 asciidoctorj:: a task extension which allows a task to extend of override global configuration for Asciidoctor tasks.
   This allow extensive flexibility. Any thing that can be configured in the global `asciidoctorj` extension can also be configured here.
 attributes:: A shortcut for `asciidoctorj.attributes`.
+baseDir:: Base directory for asciidoctor jail.
```

Comment:
"jail" ?

Reply:
Maybe need a better word, but basically Asciidoctor cannot access anything outside a jailed base directory unless the safety mode is not `UNSAFE`.

Reply:
Updated the text.